globalconf.h

```diff
@@ -111,6 +112,10 @@ typedef struct
     bool have_randr_15;
     /** Do we have a RandR screen update pending? */
     bool screen_refresh_pending;
+    /** Should screen be created before rc.lua? */
```

Comment:
screenS (same thing in the next line for the comment for `ignore_screens`)

Also, append `rc.lua is loaded` (or else it sounds like the file itself is created)

Reply:
done